### PR TITLE
New API version to Microsoft.Security 2021-11-01

### PR DESCRIPTION
**Pull request by nitsi**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
Tomarrow.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  As soon as ARM deploy the manifest changes
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

Validation tool https://github.blog/changelog/2022-02-22-sunset-notice-deprecated-teams-api-endpoints/
Got an approval for the Breaking Change from Jeffrey Richter on the weekly office hours, I'll ask him to sign it.

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ruowan** [comment]
You are adding new apis, your PR need arm review

**Comment by mentat9** [comment]
@nitsi - A couple of things before your PR is ready for ARM review:

1. Please study the materials described in the submission form and attest to that by checking the box:

**[ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.**

2. Please create a new PR following this requirement (and check the box when you submit):

**[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).**

Note: if you don't want to use OpenAPIHub to set up your new API version PR, please create it manually by copying the current API version files [unmodified] to the new API version folder and commit that as your first commit of the PR. Apply other changes in subsequent commits.

**Comment by nitsi** [comment]
@mentat9 I added the required commits to this PR as the first commits as asked.
The reason I didn't used OpenAPIHub to open this PR was because of a downtime that day (Validation tool https://github.blog/changelog/2022-02-22-sunset-notice-deprecated-teams-api-endpoints/).

I reread the documentation (this is not my first PR, but just in case there is something new).

File: specification/security/resource-manager/Microsoft.Security/stable/2021-11-01/alerts.json [hunk 1]
```diff
@@ -0,0 +1,1120 @@
...
+          {
+            "$ref": "#/parameters/AlertSimulatorRequestBody"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "original-uri"
+        },
+        "responses": {
+          "204": {
```
Comment (mentat9) [inline specification/security/resource-manager/Microsoft.Security/stable/2021-11-01/alerts.json:566]: >204 [](http://example.com/codeflow?start=11&length=3)

Your previous status codes were correct, the change is incorrect. The RPC doesn't allow 204 from POST, and 202 is required for async response.

Comment (nitsi) [reply specification/security/resource-manager/Microsoft.Security/stable/2021-11-01/alerts.json:562]: Thank you for this comment, this is not an aync operation and the result is without content.
We wish to return 204.
@mentat9

**Comment by nitsi** [comment]
@ruowan, can we marge?

**Comment by nitsi** [comment]
@ruowan it seems there is a problem with the Avocado check:

<html>
<body>


Rule | Message | My Replay
-- | -- | --
❌ NOT_LATEST_API_VERSION_IN_DEFAULT_TAG | The default tag does not contains the latest API version. Please make sure the latest api version swaggers are in the default tag.readme: specification/security/resource-manager/readme.mdjson: Microsoft.Security/preview/2020-01-01-preview/secureScore.json | using **stable** version 2020-01-01 which is newer
❌ MISSING_APIS_IN_DEFAULT_TAG | The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.readme: specification/security/resource-manager/readme.mdjson: Microsoft.Security/preview/2017-08-01-preview/pricings.json | using **stable** version 2022-03-01, it is not missing
❌ MISSING_APIS_IN_DEFAULT_TAG | The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.readme: specification/security/resource-manager/readme.mdjson: Microsoft.Security/preview/2021-08-01-preview/assignments.json | There is a suppression on that, it was not ignored until now, why?
❌ MISSING_APIS_IN_DEFAULT_TAG | The default tag does not contain all APIs in this RP. Please make sure the missing API swaggers are in the default tag.readme: specification/security/resource-manager/readme.mdjson: Microsoft.Security/preview/2021-08-01-preview/standards.json | There is a suppression on that, it was not ignored until now, why?


</body>
</html>

**Review by ruowan — APPROVED** [review]

**Comment by ruowan** [comment]
Thanks for feedback. I add approvedAvocado label.

**Review by yibirnba — APPROVED** [review]

**Pull request merged**